backends/tfhe-cuda-backend/cuda/src/utils/helper_multi_gpu.cuh

```diff
@@ -100,9 +100,13 @@ void multi_gpu_scatter_lwe_async(cudaStream_t const *streams,
                                  uint32_t const *gpu_indexes,
                                  uint32_t gpu_count, std::vector<Torus *> &dest,
                                  Torus const *src, Torus const *h_src_indexes,
-                                 bool is_trivial_index, uint32_t num_inputs,
-                                 uint32_t lwe_size) {
+                                 bool is_trivial_index,
+                                 uint32_t max_active_gpu_count,
+                                 uint32_t num_inputs, uint32_t lwe_size) {
 
+  if (max_active_gpu_count < gpu_count)
```

Comment:
isn't the check backward here ?

Reply:
you mean you would have written
if (!max_active_gpu_count >= gpu_count)
?

Reply:
ah no sorry, I was thinking in terms of assert 😵‍💫